packages/marshaller/src/ddl2/dashboard.ts

```diff
@@ -321,7 +342,7 @@ export class Dashboard extends ChartPanel {
             const dashboard = new Dashboard(ec)
                 .target(target)
                 .hideSingleTabs(true)
-                .titleVisible(false)
+                .titleVisible(true)
```

Comment:
Is this correct?

Reply:
I believe so, but I'm not immediately sure why that was necessary. I'll investigate

Reply:
@GordonSmith it looks like this change wasn't needed. I've removed it.